pos_order_return/models/pos_order.py

```diff
@@ -87,9 +86,10 @@ def partial_refund(self, partial_return_wizard):
         for wizard_line in partial_return_wizard.line_ids:
             qty = -wizard_line.qty
             if qty != 0:
-                copy_line = wizard_line.pos_order_line_id.copy()
+                copy_line = wizard_line.pos_order_line_id.copy(
+                    {'order_id': new_order.id}
+                )
                 copy_line.write({
-                    'order_id': new_order.id,
                     'returned_line_id': wizard_line.pos_order_line_id.id,
                     'qty': qty,
                 })
```

Comment:
Doesns't it make sense to add all the fields in `copy()`, and avoid calling `write()` after?

Reply:
yes , it make sense i have move all field to copy